### DISCUSSION TITLE
4.4.5 version up and bug in create persist part

### DISCUSSION THREAD

**Discussion by 253eosam** [pmndrs/zustand#2171] [Q&A] [answered]
Hello, 

yesterday, zustand is version up to 4.4.5.
I was deploying to dev environment. then show me it persist method error.

like this
```
'StateCreator<UserInfoStoreType, [], [["zustand/persist", unknown]]>' type argument is 'StateCreator<UserInfoStoreType, [], [never, unknown][]>
``` 

It went well again because I went down to version 4.4.4 for unknown reasons.

**Comment by dbritto-dev** [comment, marked as answer]
@253eosam yes, here's the fix -> https://github.com/pmndrs/zustand/pull/2170. BTW, would you mind trying the local instructions (https://ci.codesandbox.io/status/pmndrs/zustand/pr/2170/builds/434064)?

**Comment by dai-shi** [comment]
See also #2163
